examples/ml-fraud-detection/akka-streams/src/main/scala/cloudflow/examples/frauddetection/utils/FraudModelReader.scala

```diff
@@ -0,0 +1,61 @@
+package cloudflow.examples.frauddetection.utils
+
+import modelserving.model.{ ModelDescriptor, ModelType }
+import cloudflowx.ingress.ByteArrayReader
+
+/**
+ * Provides an infinite stream of wine records, repeatedly reading them from
```

Comment:
Copy paste doc issue

Reply:
The whole class was no longer needed and was deleted